src/entity/active_enum.rs

```diff
@@ -390,8 +390,8 @@ mod tests {
 
                 assert_eq!($ident::db_type(), ColumnType::$col_def.def());
 
-                assert_eq!(format!("{}", $ident::Big), "1");
-                assert_eq!(format!("{}", $ident::Small), "0");
+                assert_eq!(format!("{}", $ident::Big), "'Big'");
+                assert_eq!(format!("{}", $ident::Small), "'Small'");
```

Comment:
Currently the fallback enums doesn't pass this test as it weirdly doesn't have the single quotes 
Will try to find the reason behind it.

Reply:
We expect no single quote:

```rust
assert_eq!(format!("{}", Enum::Big), "Big");
```